prov/efa/src/efa_ep.c

```diff
@@ -93,11 +93,15 @@ static int efa_ep_modify_qp_state(struct efa_qp *qp, enum ibv_qp_state qp_state,
 	if (attr_mask & IBV_QP_QKEY)
 		attr.qkey = qp->qkey;
 
+	/* Number of retries on the firmware level.*/
+	if (attr_mask & IBV_QP_RNR_RETRY)
+		attr.rnr_retry = 1;
```

Comment:
I'd probably set this to three, not sure one retry is enough. Also can you add to your comment what 0 (no firmware retries) and 7 (firmware retries indefinitely) means?